### ISSUE TITLE
CSV Save Dialog Start Directory

### ISSUE THREAD

**Issue by Jay4242** [HenryLoenwind/sdcompare#3]
Currently, when I click 'save' for the CSV the window starts off in my home directory.
Is there a way to change that to the Source directory?  Or maybe Target Folder would make sense.  Or if there's a way for it to remember what the last directory was?
Is that something other people would want?

**Comment by HenryLoenwind** [comment]
Yes, the start folders for the various save/load dialogues are all over the place. I plan to clean that up when I implement persistent settings.
